src/js/vre/chapter36/config/form.js

```diff
@@ -258,6 +289,226 @@ const formConfig = {
     additionalInformation: {
       title: 'Additional Information',
       pages: {
+        additionalInformationVeteran: {
+          title: 'Additional Information',
+          path: 'additional-information',
+          depends: {
+            'view:isVeteran': true
+          },
+          uiSchema: {
+            previousBenefitApplications: previousBenefitApplicationsUI
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              previousBenefitApplications: {
```

Comment:
Maybe an array of [enum](https://stackoverflow.com/questions/30924271/correct-way-to-define-array-of-enums-in-json-schema)?